integration-tests/README.md

```diff
@@ -0,0 +1,106 @@
+# Integration tests for Oracle WebLogic Server Kubernetes Operator
+This documentation describes the procedure to run integration tests for the WebLogic Server Kubernetes Operator locally in the Oracle Linux environment. The tests are written in the JUnit5 framework and driven by maven profile.
+
+# Directory Configuration and Structure
+Directory structure of integration test source is distributed as follows 
+```
+weblogic-kubernetes-operator/integration-tests - location of module pom.xml  
+weblogic-kubernetes-operator/integration-tests/src/test/java/oracle/weblogic/kubernetes - integration test(JUnit5) classes and utility classes  
+weblogic-kubernetes-operator/integration-tests/src/test/resources - properties, YAML files and other bash scripts
+```
+
+# How does integration test infrastructure works
+ - Build weblogic kubernetes operator image from the downloaded branch.
+ - Download the latest version of WebLogic Deploy Tooling (WDT) and WebLogic Image Tool (WIT)
+ - Pull the specified version of WebLogic Image from container-registry.oracle.com (OCR) or phx.ocir.io (OCIR) 
+ - Build a simple WebLogic domain in image with a dynamic WebLogic cluster and a sample web application using Model-In-Image model.
```

Comment:
- Create domain resource and make sure the WebLogic domain is up and running.

Reply:
done